faithcomesbyhearing/dbp
# FCBHDBP-562 Endpoint issue: Inconsistent order of records returned in download/list endpoint for ENGGID filesets

# Description
Add consistent order to the response download/list endpoint.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-562

## How Do I QA This
- You should run the following postman test on DEV and it have to pass:

https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-fddf07fc-7dc9-491c-a8e3-1db669a96dcf

- You should run [M] Download Requests postman test and they have to pass:
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/collection/12519377-3a68ded7-3a3d-40a3-9feb-93ce491311a8?action=share&creator=17122915
